docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -0,0 +1,210 @@
+          AdvSendAdvert on;
+          AdvManagedFlag on;
+          prefix 2001:db8:2::/64
+          { 
+             AdvValidLifetime 14300; 
+             AdvPreferredLifetime 14200; 
+          };
+        };
+9. Start `radvd` service:
+
+        $ systemctl start radvd
+</details>
+
+## Building and flashing nRF Connect Lock Example Application
+
+Read [nRF Connect Lock Example Application README](../../examples/lock-app/nrfconnect/README.md) to learn how to build and flash the lock firmware onto nRF52840 DK.
+
+## Building and installing Android CHIPTool
+
+<details>
+<summary>Click here to expand</summary>
+<br>
+
+1. Read [Android CHIPTool README](../../src/android/CHIPTool/README.md) to learn how to build the application for your smartphone.
+2. Install `android-tools-adb` package:
+
+        $ sudo apt install android-tools-adb
+3. Enable **USB debugging** on the smartphone by following the [article](https://developer.android.com/studio/debug/dev-options). If **Install via USB** option is present on your Android version, turn it on, too.
+4. Plug the smartphone into a USB port on your PC.
+5. Run the following command with _chip-dir_ replaced with CHIP source directory to install the application:
+
+        $ adb install -r chip-dir/src/android/CHIPTool/app/build/outputs/apk/debug/app-debug.apk
+6. Navigate to Android Settings and grant __Camera__ and __Location__ permissions to CHIPTool.
+</details>
+
+## Commissioning
+
+<details>
+<summary>Click here to expand</summary>
+<br>
+To prepare the accessory device for commissioning:
+
+1. Attach to the device [CLI](nrfconnect_examples_cli.md) to get access to application logs.
+2. Hold **Button 1** for more than 6s to trigger factory reset of the device.
+3. Find a message similar to the one below in application logs:
+
+        I: 666[SVR] Copy/paste the below URL in a browser to see the QR Code:
+                https://dhrishi.github.io/connectedhomeip/qrcode.html?data=CH%3AI34DV%2A-00%200C9SS0
+4. Open the URL in a web browser to have onboarding QR code generated.
+5. Push **Button 4** on the device to start BLE advertising.
+
+To commission the accessory device onto the Thread network created in [Configuring PC as a Thread border router](#Configuring-PC-as-a-Thread-border-router) chapter:
+
+1. Enable **Bluetooth** and **Location** services on your smartphone.
+2. Connect the smartphone to the Wi-Fi Hotspot created in [Configuring PC as a Thread border router](#Configuring-PC-as-a-Thread-border-router) chapter.
+3. Open CHIPTool application.
+4. Tap **SCAN QR CODE** button and scan the onboarding QR code.
+5. Tap **Rendezvous over BLE** button to initiate the commissioning procedure. You should see a few pop-up messages appearing as the commissioning progresses. A message "Secure channel established. Provisioning" indicates that a secure channel has been established successfully and the accessory device has been sent Thread provisioning data.
+</details>
+
+## Sending ZCL commands
```

Comment:
Wow, where does ZCL come into play here? What is this? What is its purpose?

Reply:
I'll replace it with "CHIP mesages", ok?